web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1274,12 +1274,20 @@ export const formatters: Formatters = {
   [eventCodes.SAML_IDP_AUTH_ATTEMPT]: {
     type: 'saml.idp.auth',
     desc: 'SAML IdP authentication',
-    format: ({ user, success }) => {
+    format: ({
+      user,
+      success,
+      service_provider_entity_id,
+      service_provider_shortcut,
+    }) => {
       const desc =
         success === false
           ? 'failed to authenticate'
           : 'successfully authenticated';
-      return `User [${user}] ${desc} with the SAML IdP`;
+      const id = service_provider_entity_id
+        ? `entity ID [${service_provider_entity_id}]`
+        : `shortcut [${service_provider_shortcut}]`;
```

Comment:
Would this actually result in a rendered message like `User [alice] successfully authenticated to shortcut [xyz] with the SAML IdP`? If so, I think "shortcut" and "entity ID" are a bit confusing. Maybe just say `User [alice] successfully authenticated to SAML service provider [xxx]` and use either entity id or shortcut depending on what is set?

Reply:
Sounds good!